src/components/templates/Audience/Audience.tsx

```diff
@@ -50,6 +50,10 @@ const VIDEO_MIN_WIDTH_IN_SEATS = 8;
 // We should keep the 16/9 ratio
 const VIDEO_MIN_HEIGHT_IN_SEATS = VIDEO_MIN_WIDTH_IN_SEATS * (9 / 16);
 
+// Always have an odd number of columns.
```

Comment:
Otherwise, it ruins the display.

Reply:
> it ruins the display.

That's... not really a useful answer...

But I will expand and be more explicit/verbose/articulate in my questions.

**Why** does it ruin the display? What aspect of the code as it exists relies on there being an odd number of rows. Is it that a fundamental necessity, or is there some way that we can change that to work for any number of rows? Etc.

This is the level of detail in my question, and the solution to which (assuming we can't make it work for anything) that I want to see captured/summarised in the requested comment here.

"Just because" isn't good enough, and that's essentially what the comment is saying now.

Reply:
Not an odd number of rows, but an odd number of columns. That's because of this visual `fire line` or `separator`, located just in the middle of the screen.

Reply:
So if we didn't need to have that 'visual fireline' separator, it would work with both even/odd?

I feel it would.. because:

- if we have an odd number, then we conceptually have 1 'middle line' and then equal seats either side, so it remains 'balanced'
- if we have even, we have no 'middle line' and still have equal seats either side, so it should still remain balanced

The only 'potential issue' I can see is if we were exactly aligning the 'center embeddable' based on the columns, but since we aren't doing that (as it is just centered using flex/etc) then there shouldn't be a problem here.

---

So, based on the above, I think the 'advice' required here in the comments is that we only require an odd number of columns when we want that 'fire line'/separator.

We should also create a boolean/similar (can just be a const for now, but eventually it would make sense to move it into venue config within firestore) for whether to show the fire line or not. That way, in future, we can add it to venue config/admin UI and then base our data validations on it, giving proactive/useful error messages to 'guide' the user in selecting the right value, based on the actual constraints.

From my perspective, the boolean/comment are in-scope for this PR, the admin UI/proactive hints/etc may well be OOS.

Reply:
(backlogged this in https://github.com/sparkletown/internal-sparkle-issues/issues/573, but still worth doing the 'in-scope' parts I described above)

Reply:
`We should also create a boolean/similar (can just be a const for now, but eventually it would make sense to move it into venue config within firestore) for whether to show the fire line or no`

I have a better idea. Rather than creating an additional toggler, I could just show it, if the number of columns is odd and not show it, when it's even. WDYT?

Reply:
I feel like it's sort of only really useful for burns/similar, not necessarily general events, which is why I was thinking having it as an explicit toggle that can be turned off. Having it appear/disappear based on even/odd feels like 'confusing magic' from an end-user perspective, and is more likely to feel like a bug/like something is broken IMO.

Reply:
added as OOS in https://github.com/sparkletown/sparkle/pull/1251#discussion_r592830551